build/linux/installer/conf/fluent-bit.conf

```diff
@@ -44,3 +44,16 @@
 #${MultilineEnabled}    Match oms.container.log.la.*
 #${MultilineEnabled}    multiline.key_content log
 #${MultilineEnabled}    multiline.parser #${MultilineLanguages}
+
+#${KubernetesFilterEnabled}[FILTER]
```

Comment:
we should make these work on geneva path also as internal teams are asking for this.

Reply:
added for geneva